osversion/platform_compat_windows.go

```diff
@@ -0,0 +1,70 @@
+package osversion
+
+import (
+	"strconv"
+	"strings"
+)
+
+// List of stable ABI compliant
+// ltsc releases
+// Note: List must be sorted
+var compatLtscReleases = []uint{
+	V21H2Server,
+}
+
+// Checks for stable ABI compliant OS versions.
+// Every release after WS 2022 will support the previous
+// ltsc container image. Stable ABI is in preview mode
+// for windows 11 client.
+// Refer: https://learn.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility?tabs=windows-server-2022%2Cwindows-10#windows-server-host-os-compatibility
+func CanRun(hostOSVersion, guestOSVersion string) bool {
+	// HPC containers do not have the hostOSVersion set
+	if hostOSVersion == "" {
```

Comment:
Shouldn't this be `guestOSVersion`? With HPC we still know the host version, it's just the container image that may not have one set.

Reply:
I actually kind of feel that we should not have this HPC specific handling in here at all, and that should just live in the containerd platform matcher. Then this function can focus just on comparing two versions, and not what the implications may be if one of them is empty (which can then just be an error case).